tests/phpunit/tests/blocks/renderCommentTemplate.php

```diff
@@ -194,25 +194,27 @@ function test_rendering_comment_template() {
 			)
 		);
 
-		$this->assertEquals(
-			'<ol class="wp-block-comment-template"><li id="comment-' . self::$comment_ids[0] . '" class="comment even thread-even depth-1"><div class="has-small-font-size wp-block-comment-author-name"><a rel="external nofollow ugc" href="http://example.com/author-url/" target="_self" >Test</a></div><div class="wp-block-comment-content">Hello world</div></li></ol>',
-			$block->render()
+		$this->assertSame(
+			str_replace( array( "\n", "\t" ), '', '<ol class="wp-block-comment-template"><li id="comment-' . self::$comment_ids[0] . '" class="comment even thread-even depth-1"><div class="wp-block-comment-author-name"><a rel="external nofollow ugc" href="http://example.com/author-url/" target="_self" >Test</a></div><div class="wp-block-comment-content"><p>Hello world</p></div></li></ol>' ),
+			str_replace( array( "\n", "\t" ), '', $block->render() )
 		);
 	}
 
 	/**
-	 * Test rendering 3 nested comments:
+	 * Test rendering nested comments:
 	 *
 	 * └─ comment 1
 	 *    └─ comment 2
 	 *       └─ comment 3
+	 *       └─ comment 4
+	 *    └─ comment 5
```

Comment:
```suggestion
```

Reply:
Ah, looks like this didn't make it into `trunk`: https://github.com/WordPress/wordpress-develop/blob/a913b7312db9ea68bd86e9658ec69a97473bfb30/tests/phpunit/tests/blocks/renderCommentTemplate.php#L206-L210

Here's what the structure should look like: https://github.com/WordPress/gutenberg/blob/d0b600f44406d48fa8950fb3d519630aaf6bfea6/phpunit/class-block-library-comment-template-test.php#L149-L152